### PR TITLE
FIX(pipewire): Do not initialize stream with 0 channels

### PR DESCRIPTION
**Pull request by vimpostor**
### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

**Comment by vimpostor** [comment]
I see you have already branched for the 1.4 release. Should I backport to the 1.4.x branch?

**Comment by davidebeatrici** [comment]
Yes, thanks!

**Review by davidebeatrici — APPROVED** [review]

File: src/mumble/Settings.cpp [hunk 1]
```diff
@@ -413,6 +413,9 @@ Settings::Settings() {
 	qsALSAInput  = QLatin1String("default");
 	qsALSAOutput = QLatin1String("default");
 
+	pipeWireInput = 1;
+	pipeWireOutput = 2;
```
Comment (davidebeatrici) [inline src/mumble/Settings.cpp:417]:
```suggestion
	pipeWireInput  = 1;
	pipeWireOutput = 2;
```

**Comment by Krzmbrzl** [comment]
@davidebeatrici don't forget to backport PRs like this to 1.4.x
Otherwise it won't be in the stable release :)

**Pull request merged**